tests/wdio.conf.js

```diff
@@ -0,0 +1,156 @@
+exports.config = {
```

Comment:
I can remove all the comments. This is a standard file generated by the configuration generator. I thought the comments are useful and make it easier to understand.